### DISCUSSION TITLE
Welcome to ExtensibilityTemplatePack Discussions!

### DISCUSSION THREAD

**Discussion by madskristensen** [VsixCommunity/ExtensibilityTemplatePack#5] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yannduran** [comment]
Hey everyone!

Is there any reason why these templates couldn't be included with the _Community Toolkit_ itself? As in **_in_** the Nuget package, and installed/copied to either the appropriate Visual Studio folder, or in the user's Documents folder (where templates are also searched for).

Most of the templates in this extension don't make any sense if the _Community Toolkit_ isn't being used, so why not (if it's technically possible) have all of the _Community_ templates together with the toolkit itself?

Just a thought.
